hibernate-core/src/main/java/org/hibernate/boot/model/source/internal/hbm/ModelBinder.java

```diff
@@ -2547,22 +2534,22 @@ private void bindProperty(
 							mappingDocument.getOrigin()
 					);
 				}
-				if ( generationTiming.isNotNever() ) {
-					property.setValueGeneratorCreator(context -> new GeneratedValueGeneration( generationTiming ) );
+				if ( timing != GenerationTiming.NEVER ) {
```

Comment:
I find it odd that you go out of the way to define simplified checks for all other cases (`#includesInsert`, etc) but here directly check equality on the enum.  Seems like this should be consistent

Reply:
aaaaaaah good point. I added `isNotNever()` in an earlier PR, because I was using it in lots of places. Not anymore.

So I was planning on removing both `isNotNever()` and `isAlways()` because after this change they were not needed: 

- `isNotNever()` was now only used in that one place, and
- `isAlways()` is now unused (but then I actually forgot to remove it!)

I should either:

- remove `isAlways()`, or
- put back `isNotNever()`.

Your call, I don't care.